src/Cli/Program.cs

```diff
@@ -106,7 +107,62 @@ public static int Main(string[] args)
                     {
                         cliLogger.LogError("Failed to start the engine.");
                     }
-                })
+                }))
+                .WithParsed((Action<ExportOptions>)(options =>
+                {
+                    StartOptions startOptions = new(false, LogLevel.None, false, options.Config!);
+
+                    CancellationTokenSource cancellationTokenSource = new();
+                    CancellationToken cancellationToken = cancellationTokenSource.Token;
+
+                    Task server = Task.Run(() =>
+                    {
+                        _ = ConfigGenerator.TryStartEngineWithOptions(startOptions);
+                    }, cancellationToken);
+
+                    if (options.GraphQL)
+                    {
+                        int retryCount = 5;
+                        int tries = 0;
+
+                        while (tries < retryCount)
+                        {
+                            try
+                            {
+                                HttpClient client = new( // CodeQL[SM02185] Loading internal server connection
+                                    new HttpClientHandler { ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator }
+                                )
+                                { BaseAddress = new Uri("https://localhost:5001/graphql") };
```

Comment:
this /graphql endpoint will only work if not set otherwise in runtime configuration

Reply:
so to avoid having to read the config in this CLI option, would an argument like 'GraphQLEndpoint' make sense? Can default it to /graphql if not provided.
`./Microsoft.DataApiBuilder export --graphql -o ./schemas -endpoint '/mygqlendpoint'`

Reply:
ideally, since the config is provided, we should be intelligent enough to determine where to invoke the graphql schema request to, without needing additional input from the developer.

Reply:
Good point that we should be following with what they set in the config. I'll look at how easy it is to load the config within the exporter.